src/main/java/org/terasology/navgraph/BitMap.java

```diff
@@ -20,12 +20,17 @@
 
 /**
  * @author synopia
+ * Representation of a any region or floor
  */
 public class BitMap {
     public static final int KERNEL_SIZE = 3;
     public static final float SQRT_2 = (float) Math.sqrt(2);
     BitSet map;
 
+    /**
+     * Creates a new Bitmap
+     */
+
```

Comment:
Get rid of this extra line

Reply:
I'm a bit confused about the newline comments. Is it line 33 on this?